code/modules/mining/machine_stacking.dm

```diff
@@ -4,6 +4,7 @@
 	name = "stacking machine console"
 	icon = 'icons/obj/machines/mining_machines.dmi'
 	icon_state = "console"
+	desc = "Controls a stacking machine. In theory."

```

Comment:
comma after machine instead of period

Reply:
It's supposed to be a read in a tone like "This controls a stacking machine. [pause] In theory" like in a mysterious way almost. I could use an ellipse to convey that better maybe

Reply:
yeah go with that